docs/Developers/Newcomers_Guide.rst

```diff
@@ -23,14 +23,15 @@ following tasks, you will become a developer at coala:
   - merge a ``difficulty/low`` Pull Request
   - review at least a ``difficulty/low`` or higher Pull Request
 
-When you ran coala on a project, please fill our
-`usability survey <http://coala.io/usability>`_. Once you got your first Pull
-Request merged successfully, fill in our
-`survey form <http://coala.io/newform>`_. With that you can help us making your
-experience better!
-
-Once you have achieved all these, just ask for being promoted on the chat and
-provide links to your reviews and merged Pull Requests. Then, you will be able
+
+When you have run coala on a project, please fill out our
+`usability survey <http://coala.io/usability>`_. Once you have successfully
+merged your first Pull Request, fill in our
+`survey form <http://coala.io/newform>`_. That will help us make your
+experience better.
+
+Once you have achieved these steps, just ask to be promoted on the chat and
```

Comment:
IMHO: `achieved` -> `completed `
because *goals* are achieved, steps should be completed. 😄